AssetsPickerViewController/Classes/Photo/Controller/AssetsPhotoViewController.swift

```diff
@@ -739,8 +758,14 @@ extension AssetsPhotoViewController: AssetsManagerDelegate {
         }
     }
     
-    public func assetsManager(manager: AssetsManager, updatedAlbums albums: [PHAssetCollection], at indexPaths: [IndexPath]) {}
-    public func assetsManager(manager: AssetsManager, reloadedAlbum album: PHAssetCollection, at indexPath: IndexPath) {}
+    public func assetsManager(manager: AssetsManager, updatedAlbums albums: [PHAssetCollection], at indexPaths: [IndexPath]) {
+        AssetsManager.shared.fetchAssets(isRefetch: true) { (_) in
+            self.collectionView.reloadData()
+        }
+    }
+    public func assetsManager(manager: AssetsManager, reloadedAlbum album: PHAssetCollection, at indexPath: IndexPath) {
+        
+    }
```

Comment:
Same. Empty method. Can we remove it?

Reply:
This class is conforming to 'AssetsManagerDelegate' so these delegate methods need to be implemented. otherwise I have to write and extension for 'AssetsManagerDelegate' and make empty implementations for those particular methods.

Reply:
OK, I see, then we can keep them.